pegasus/cache/i18n/en-US.yml

```diff
@@ -281,7 +281,7 @@
   inspirational_videos_more: "More <a href=\"https://www.youtube.com/playlist?list=PLzdnOPI1iJNfpD8i4Sx7U0y2MccnrNZuP\">inspirational videos</a> by role models and celebrities"
   playlab_choose_theme: "Choose your Play Lab theme"
   playlab_classic_title: "Classic"
-  playlab_gumball_title: "Gumball"
+  playlab_gumball_title: "The Amazing World of Gumball"
```

Comment:
@caleybrock for a later PR -- we should make sure it's the full name everywhere. /s/gumball has a different title.